transcriptic/cli.py

```diff
@@ -672,6 +672,12 @@ def format_cmd(manifest):
     default=None,
     help="Delay in minutes at which the given protocol should start (at the earliest).",
 )
+@click.option(
+    "--time-constraints-are-suggestion",
+    "-tc-suggestion",
+    help="If set, the time constraints will be considered only suggestion.",
+    is_flag=True,
+)
```

Comment:
I think we should include the default value you set in this [PR](https://github.com/strateos/lab/pull/1496/files#diff-515485d184b82bc0157ab6ea7b86ea2422e5cfb280b59334b881a99d05b80490R83-R88)

Reply:
Like it is a flag, I think the value will be false if the flag is not set, or true if it is set. But I haven't done a full pipeline test, I will do.

Reply:
should be "only a suggestion"

Reply:
Got it, makes sense from the comments on the other PR.